tomcat-gcp-lib/src/main/java/com/google/cloud/runtimes/tomcat/session/DatastoreManager.java

```diff
@@ -91,10 +95,13 @@ protected synchronized void startInternal() throws LifecycleException {
   @Override
   public Session findSession(String id) throws IOException {
     log.debug("Datastore manager is loading session: " + id);
+    Tracer tracer = Trace.getTracer();
```

Comment:
I thought tracing is optional, and user should be able to opt-out of StackDriver trace, if they want to. 
Isn't that the `gcp.distributed-sessions.enableTrace` is for?

Reply:
By default a trace service which ignore all the traces is instancied (`NoTraceService` in the [Trace implementation](https://github.com/GoogleCloudPlatform/cloud-trace-java/blob/master/sdk/service/src/main/java/com/google/cloud/trace/Trace.java)).
Enabling the module `stackdriver-trace` replace this service with a GrpcTraceService, which send the traces to Stackdriver.

However I forgot to check for `gcp.distributed-sessions.enableTrace` in this class, I will correct that.